tests/framework/helpers/HtmlTest.php

```diff
@@ -671,11 +670,11 @@ public function testRemoveCssClass()
     {
         $options = ['class' => 'test test2 test3'];
         Html::removeCssClass($options, 'test2');
-        $this->assertEquals(['class' => 'test test3'], $options);
+        $this->assertEquals(['class' => ['test', 2 => 'test3']], $options);
```

Comment:
Doesn't look good to me. Return type should be compatible if you want it to be Yii 2.0.x-compatible.